plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java

```diff
@@ -543,6 +559,20 @@ public String getEffectivePom(String projectPath) {
     return result;
   }
 
+  private void reComputeDiagnostics(String pomPath) {
+    String pomUri = prefixURI(pomPath);
+    Type type = new TypeToken<PublishDiagnosticsParams>() {}.getType();
+    PublishDiagnosticsParams diagnostics =
+        doGetOne(Commands.RECOMPUTE_POM_DIAGNOSTICS, singletonList(pomUri), type);
```

Comment:
As said in the companion PR: use the normal diagnostics publishing mechanism instead of notifying here.

Reply:
Fixed